sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -761,6 +762,25 @@ dynamic parameter for the buffer can be registered with all the nodes which
 use the buffer as a parameter. Then a single `dynamic_parameter::update()` call
 will maintain the graphs data dependencies.
 
+===== Whole Graph Update [[whole-graph-update]]
+
+A graph in the executable state can have all of its nodes inputs & outputs
```

Comment:
Wondering if this "inputs & outputs" terminology we use throughout the new wording is too restrictive. We can update the nd-range as well, and it's not clear that's covered by "inputs & output"

Reply:
Agreed. Taking into account that dependencies could be also considered as inputs and outputs, we might want to be more specific here.

Reply:
I've updated this to be more specific and point to the previously established list of what can be updated on nodes.

Reply:
Thanks for updating this here, the phrase "inputs & outputs" is still mentioned a few other times in the API description and was thinking of being more specific in those locations as well so that there are no occurrences of this phrase

Reply:
I've generally removed usage of "inputs & outputs" now, replacing it with a more generic "parameters" where appropriate, and rewording some of the surrounding texts.